### PR TITLE
Fix #8 - Add additional authentication method support for v1 API

### LINKED ISSUE #8: LDAP authentication support
According to the Authentication documentation, it says it supports LDAP.  I don't see an option on how to use the module while using an LDAP account.  I took at look at the Python module and only see support for Shared Auth and CyberArk Auth.   

https://github.com/cyberark/ansible-security-automation-collection/blob/master/docs/cyberark_authentication.md

### PR DESCRIPTION
**Pull request by infamousjoeg**
Added a new `auth_type` module variable to accept `cyberark`, `ldap`, `windows` authentication methods.

The default value of `auth_type` is `cyberark`.

### CODE REVIEW

**Comment by AndrewCopeland** [comment]
I use LDAP authentication when using Cyberark so this would be helpful.

**Comment by infamousjoeg** [comment]
@JimmyJamCABD will these receive a review?

File: plugins/modules/cyberark_authentication.py [hunk 1]
```diff
@@ -311,6 +325,7 @@ def main():
         "username": {"type": "str"},
         "password": {"type": "str", "no_log": True},
         "new_password": {"type": "str", "no_log": True},
+        "auth_type": {"default": "cyberark", "type": "str"}
```
Comment (murrahjm) [inline plugins/modules/cyberark_authentication.py:328]: need a comma at the end here

**Comment by infamousjoeg** [comment]
updated @murrahjm

**Comment by infamousjoeg** [comment]
LDAP Authentication was added during the refactor to v2 API endpoints.  Closing this PR...

**Pull request closed without merging**